code/modules/mob/living/carbon/xenomorph/evolution.dm

```diff
@@ -16,14 +16,27 @@
 	set desc = "Change into another caste in the same tier."
 	set category = "Alien"
 
-	if(world.time - (GLOB.key_to_time_of_caste_swap[key] ? GLOB.key_to_time_of_caste_swap[key] : -INFINITY) < 9000) //casteswap timer, 15 minutes
+	if(world.time - (GLOB.key_to_time_of_caste_swap[key] ? GLOB.key_to_time_of_caste_swap[key] : -INFINITY) < (15 MINUTES))
 		to_chat(src, span_warning("Your caste swap timer is not done yet."))
 		return
 
 	SStgui.close_user_uis(src, GLOB.evo_panel)
 	ADD_TRAIT(src, TRAIT_CASTE_SWAP, TRAIT_CASTE_SWAP)
 	GLOB.evo_panel.ui_interact(src)
 
+/mob/living/carbon/xenomorph/verb/strain_swap()
```

Comment:
Verb ?

Reply:
We still dont have an option for this, and caste swap/evo/regress are the same so